mava/types.py

```diff
@@ -147,7 +78,7 @@ class RNNEvalState(NamedTuple):
     episode_return: chex.Array
 
 
-MavaState = TypeVar("MavaState", LearnerState, RNNLearnerState, EvalState, RNNEvalState)
+MavaState = TypeVar("MavaState")  # Types like: LearnerState | EvalState.
```

Comment:
This seems difficult to catch since the learner state in the ppo/types.py, can we make it clear?
This is not the best idea, but we can say something like "For more info on the LearnerState type, please refer to ppo/types.py."?